backend/src/org/commcare/suite/model/Callout.java

```diff
@@ -0,0 +1,100 @@
+package org.commcare.suite.model;
+
+import org.javarosa.core.model.condition.EvaluationContext;
+import org.javarosa.core.util.externalizable.DeserializationException;
+import org.javarosa.core.util.externalizable.ExtUtil;
+import org.javarosa.core.util.externalizable.ExtWrapNullable;
+import org.javarosa.core.util.externalizable.Externalizable;
+import org.javarosa.core.util.externalizable.PrototypeFactory;
+import org.javarosa.xpath.XPathParseTool;
+import org.javarosa.xpath.expr.XPathFuncExpr;
+import org.javarosa.xpath.parser.XPathSyntaxException;
+
+import java.io.DataInputStream;
+import java.io.DataOutputStream;
+import java.io.IOException;
+import java.util.Hashtable;
+import java.util.Vector;
+
+/**
+ * Created by wpride1 on 4/14/15.
+ */
+public class Callout implements Externalizable, DetailTemplate {
+
+    private String actionName;
+    private String image;
+    private final String displayName;
+    private final Hashtable<String, String> extras = new Hashtable<String, String>();
+    private final Vector<String> responses = new Vector<String>();
+
+    public Callout(String actionName, String image, String displayName) {
+        this.actionName = actionName;
+        this.image = image;
+        this.displayName = displayName;
+    }
+
+    @Override
```

Comment:
Explicit override annotations are useful because they statically check that your method signature matches the one you are overriding. Recommend using in the future